osu.Game/Localisation/MultiplayerCountdownButtonStrings.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class MultiplayerCountdownButtonStrings
+    {
+        private const string prefix = @"osu.Game.Resources.Localisation.MultiplayerCountdownButton";
+
+        /// <summary>
+        /// "Stop countdown"
+        /// </summary>
+        public static LocalisableString StopCountdown => new TranslatableString(getKey(@"stop_countdown"), @"Stop countdown");
+
+        /// <summary>
+        /// "Countdown settings"
+        /// </summary>
+        public static LocalisableString CountdownSettings => new TranslatableString(getKey(@"countdown_settings"), @"Countdown settings");
+
+        /// <summary>
+        /// "Start match in {0} minute/seconds"
```

Comment:
I was unsure how to format this XML doc, so suggestions are welcome.

Reply:
For now it should just match 1:1.